torch/nn/functional.py

```diff
@@ -349,8 +349,9 @@
       tuple `(kW,)`
     stride: the stride of the window. Can be a single number or a tuple
       `(sW,)`. Default: :attr:`kernel_size`
-    padding: implicit zero paddings on both sides of the input. Can be a
-      single number or a tuple `(padW,)`. Default: 0
+    padding: implicit zero paddings on both sides of the input. Can be a single
+      number or a tuple `(padW,)`. Should be at most half of effective kernel
```

Comment:
The half of effective kernel size constraint is mentioned in the note on 341

https://github.com/pytorch/pytorch/blob/a26cf98d9fe46ab0def470637a9feb8b871eea67/torch/nn/functional.py#L341-L342

This change is ok but could you remove the .. note:: there and in all the corresponding modules please